src/bun.js/webcore/Body.zig

```diff
@@ -598,6 +598,20 @@ pub const Value = union(Tag) {
             }
         }
 
+        if (value.as(bun.jsc.API.HTMLBundle)) |html_bundle| {
```

Comment:
If you want to implement this, then we should add *HTMLBundler as a union member of Body.Value and then have it increment / decrement a reference count. And probably throw if you try to use any of the body value members

Reply:
done!